net-libs/tox/metadata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
```

Comment:
You have no authority to prevent people from committing live ebuilds. 'I want to commit this one day in far future' doesn't help anyone, neither does pointing them to low quality overlays.